pkg/common/outcomes/unknown.go

```diff
@@ -0,0 +1,21 @@
+package outcomes
+
+var _ Outcome = (*Unknown)(nil)
+
+// Unknown is for check outcomes where the outcome isn't known.
```

Comment:
Would it be clearer to say something like "Unknown occurs when the check was unable to conclude."